src/extension.ts

```diff
@@ -461,6 +461,45 @@ function launchMetals(
       }
     });
 
+    registerCommand("metals-goto-location", args => {
```

Comment:
Why do we need to register the command? I think this will also go to the settings and will be able to be used by the user?

Reply:
not needed, removed :+1: